onnxruntime/python/tools/transformers/shape_infer_helper.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
The tricky part is that the sybolic_shape_infer.py is not placed in .. (It is in ..\tools) inside onnxruntime package. Maybe add a check with os.path.exists() then update the path accordingly.

Reply:
Got it. Thanks